src/pydata_sphinx_theme/assets/styles/content/_api.scss

```diff
@@ -92,3 +92,13 @@ table.field-list {
 dt:target {
   background-color: var(--pst-color-target);
 }
+
+.viewcode-back {
+  font-family: var(--pst-font-family-base);
+}
+
+.viewcode-block:target {
```

Comment:
Side note, I've realize this morning in one of my documenation that the coloring of targeted viewcode doesn't work at all with tango (dark theme).

Reply:
@12rambau, do you want to open a follow-up issue for that one? Thanks!